src/common/integration/urlProcessing.js

```diff
@@ -10,6 +10,7 @@ type Props = {
 };
 
 const SelectUrlBooking = (props: Props) => {
+  // Does this need to include whitelabels?
   const urlMatch = window.location.href.match(
     /.*[kiwi.com|localhost:\d*]\/.*\/account\/bookings\/(\d*)\?.*$/,
```

Comment:
Not sure if this needs to change? @jaroslav-kubicek

Reply:
I don't think this is related to the issue :slightly_smiling_face: